src/env.js

```diff
@@ -24,4 +24,25 @@ module.exports = envalid.cleanEnv(process.env, {
     default: 4001,
     desc: 'subscription port (websocket)',
   }),
+  PAYPAL_MODE: envalid.str({
+    default: 'sandbox',
+    desc: 'paypal environment sandbox | live',
+  }),
+  PAYPAL_CLIENT_ID: envalid.str({
+    default: '',
```

Comment:
Should `PAYPAL_CLIENT_ID` and `PAYPAL_CLIENT_SECRET` have a default value? Or should `npm start` fail with an error from envalid when not defined?